api/src/pcapi/sandboxes/scripts/utils/storage_utils.py

```diff
@@ -1,16 +1,21 @@
 from pathlib import Path
+import typing
 
 from pcapi.connectors.thumb_storage import create_thumb
 from pcapi.core.object_storage import store_public_object
+from pcapi.models import Model
 import pcapi.sandboxes
 from pcapi.utils.human_ids import humanize
 
 
-def store_public_object_from_sandbox_assets(folder, model, subcategoryId):  # type: ignore [no-untyped-def]
+T = typing.TypeVar("T", bound=Model)
+
+
+def store_public_object_from_sandbox_assets(folder: str, model: T, subcategory_id: str) -> T:
```

Comment:
En local ça fonctionne également avec (plus simple imo)

```suggestion
def store_public_object_from_sandbox_assets(folder: str, model: Model, subcategory_id: str) -> Model:
```

Je veux bien comprendre l’utilité du coup

Reply:
sinon c’est ok pour moi

Reply:
Yes c'est toi qui a raison
C'est un relicat d'une approche où je typais différement, pour dire à mypy que le modèle de `model` est le même que le type de retour. 
Je n'ai pas trouvé un moyen simple donc je suis revenu en arrière (mais pas assez!) 

J'update